core/unix/coredump.c

```diff
@@ -71,8 +71,7 @@
 #define PROGRAM_HEADER_NOTE_LENGTH                                                    \
     (sizeof(ELF_NOTE_HEADER_TYPE) + NOTE_OWNER_LENGTH + sizeof(struct elf_prstatus) + \
      sizeof(ELF_NOTE_HEADER_TYPE) + NOTE_OWNER_LENGTH + sizeof(elf_fpregset_t))
-#define PROGRAM_HEADER_LENGTH \
-    (sizeof(ELF_PROGRAM_HEADER_TYPE) + PROGRAM_HEADER_NOTE_LENGTH)
+#define PROGRAM_HEADER_LOAD_ALIGNMENT 0x1000
```

Comment:
Where does this come from?  Should this instead be the page size?  Is this from some documented standard?  Please add a comment.

Reply:
I've changed the code to use os_page_size().